src/app/core/services/personal-cards.service.spec.ts

```
@@ -189,16 +191,47 @@
+      spyOn(personalCardsService, 'deleteAccountPlatform').and.callThrough();
+      spyOn(personalCardsService, 'transformPersonalCardPlatformArray').and.callThrough();
+
+      const accountId = 'bacc0By33NqhnS';
+      const payload = {
+        data: {
+          id: accountId,
+        },
+      };
+
+      let response: PersonalCard;
+      personalCardsService.deleteAccount(accountId).subscribe((res) => (response = res));
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)